### PR TITLE
feat: add metric for all mobile bridge "unregistered" user drops

### LINKED ISSUE #1421: Emit detailed metrics for new drop_user on bridge 404s
We should emit a metric with detailed tag information about the bridge and its response code that caused the drop.

### PR DESCRIPTION
**Pull request by jrconlin**
* normalized metric name and tags

Closes #1421

## Description
* made the APNS user disconnect metric more generic across all mobile platforms
* normalized some tag information

## Testing

Tests included. 

## Issue(s)

Closes #1421.

### CODE REVIEW

File: autopush/web/base.py [hunk 1]
```diff
@@ -335,6 +335,14 @@ def _router_fail_err(self, fail, router_type=None, vapid=False, uaid=None):
         # reporting a not found and they're mobile.
         if exc.status_code in [404, 410] and router_type in [
                 'apns', 'fcm', 'adm']:
+            tags = self._base_tags.update({
```
Comment (Mcat12) [inline autopush/web/base.py:338]: `update` mutates the dictionary and returns `None`:
https://docs.python.org/3/library/stdtypes.html#dict.update

This is probably not the behavior you're expecting?

Comment (jrconlin) [reply autopush/web/base.py:338]: crap. You're right. I always get that wrong. Thanks!

File: autopush/web/base.py [hunk 2]
```diff
@@ -335,6 +335,14 @@ def _router_fail_err(self, fail, router_type=None, vapid=False, uaid=None):
         # reporting a not found and they're mobile.
         if exc.status_code in [404, 410] and router_type in [
                 'apns', 'fcm', 'adm']:
+            self._base_tags.update({
+                "platform": router_type,
+                "reason": "unregistered",
+            })
```
Comment (Mcat12) [inline autopush/web/base.py:341]: I'm concerned that mutating the base tags here might leak into other usages. Are these tags reused after this point, or are they recreated?

Comment (jrconlin) [reply autopush/web/base.py:341]: It's part of `_router_fail_err` which is an error handler and is one of the last stages for the request Handler.
Mutating the base tags here is fine, and expected.

File: autopush/router/adm.py [hunk 3]
```diff
@@ -191,7 +191,7 @@ def _route(self, notification, uaid_data):
             self.metrics.increment("notification.bridge.error",
                                    tags=make_tags(
                                        self._base_tags,
-                                       reason="connection_unavailable"))
+                                       reason="connection unavailable"))
```
Comment (pjenvey) [inline autopush/router/adm.py:194]: FYI this *denormalized* the reason, it's an underscore everywhere else

**Pull request merged**